components/event-sources/config/300-httpsource-crd.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 The Kyma Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: httpsources.sources.kyma-project.io
+  labels:
+    eventing.knative.dev/source: 'true'
+    kyma-project.io/crd-install: 'true'
+spec:
+  group: sources.kyma-project.io
+  scope: Namespaced
+  names:
+    kind: HTTPSource
+    plural: httpsources
+    categories:
+    - all
+    - knative
+    - eventing
+    - sources
```

Comment:
imho it would make sense to introduce a new category that contains all kyma-eventing related objects (including the application, api ... )
adding it here doesnt harm anyone but it would be a start

Reply:
Voice your wishes.

Reply:
how about `kyma-eventing`

Reply:
Wish granted.

Reply:
thx